src/gui/properties/peerlistwidget.cpp

```diff
@@ -138,6 +146,62 @@ PeerListWidget::~PeerListWidget()
     delete m_copyHotkey;
 }
 
+void PeerListWidget::displayToggleColumnsMenu(const QPoint&)
+{
+    QMenu hideshowColumn(this);
+    hideshowColumn.setTitle(tr("Column visibility"));
+    QList<QAction*> actions;
+    for (int i = 0; i < PeerListDelegate::IP_HIDDEN; ++i) {
+        QAction *myAct = hideshowColumn.addAction(headerData(i, Qt::Horizontal, Qt::DisplayRole).toString());
```

Comment:
You don't need to call `PeerListWidget::headerData()`. Just call `m_listModel->headerData()` instead.